Source/Core/InputCommon/ControllerEmu/StickGate.cpp

```diff
@@ -275,6 +281,41 @@ ReshapableInput::ReshapeData ReshapableInput::Reshape(ControlState x, ControlSta
   x -= m_center.x;
   y -= m_center.y;
 
+  // Let's just convert everything to positive then convert back afterwards
+  const int xNegMultiplier = x >= 0 ? 1 : -1;
```

Comment:
```suggestion
  const int x_neg_multiplier = x >= 0 ? 1 : -1;
```

ditto for `yNegMultiplier` and `absX` and `absY`